.gitattributes

```diff
@@ -0,0 +1 @@
+client/docs/swagger-ui/* linguist-vendored
```

Comment:
The auto-generated Javascript files in the `client/docs/swagger-ui/` ruin the language statistic of GitHub.
![image](https://user-images.githubusercontent.com/5462944/125268719-f0122600-e342-11eb-842c-f837d936dc50.png)